evm/p2p/utils.py

```diff
@@ -19,3 +21,12 @@ def roundup_16(x):
 
 def gen_request_id():
     return big_endian_to_int(os.urandom(8))
+
+
+def get_devp2p_cmd_id(msg):
+    """Return the cmd_id for the given devp2p msg.
+
+    The cmd_id, also known as the payload type, is always the first entry of the RLP, interpreted
+    as an integer.
+    """
+    return rlp.decode(msg[:1], sedes=rlp.sedes.big_endian_int)
```

Comment:
In the event that `msg == b''` this will raise a confusing error: `DecodingError: RLP string to short`

Does it make sense to special case that here or is there validation elsewhere for this.

Reply:
Peer.read_msg() cannot return an empty bytes since it will always either read/decrypt a full, valid msg, or raise an exception.